packages/fast-exit/contracts/L1PassiveFastExitManager.sol

```diff
@@ -0,0 +1,82 @@
+pragma solidity >=0.6.11;
+
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+import "arb-bridge-peripherals/contracts/tokenbridge/ethereum/gateway/L1ArbitrumExtendedGateway.sol";
+
+interface IExitLiquidityProvider {
+    function requestLiquidity(
+        address dest,
+        address erc20,
+        uint256 amount,
+        uint256 exitNum,
+        bytes calldata liquidityProof
+    ) external returns (bytes memory);
+}
+
+contract L1PassiveFastExitManager is ITradeableExitReceiver {
```

Comment:
We should probably add a note that these contracts are not intended for use in production.
A disclaimer both in the README.md of the repo and the contracts as natspec would be cool
```suggestion
/// @title A intermediate contract to help coordinate trustless fast exits
/// @notice This contract is only intended as a source of inspiration and is by no means intended for use in production
contract L1PassiveFastExitManager is ITradeableExitReceiver {
```